com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/TilingAndBinningUtilities.hlsl

```diff
@@ -231,6 +231,26 @@ uint ComputeTileBufferIndex(uint tile, uint category, uint eye)
     return ComputeTileBufferIndex(tile, category, eye, TILE_BUFFER_DIMS);
 }
 
+// Internal. Do not call directly.
+uint ComputeZBinBitArrayBufferIndex(uint zbin, uint category, uint eye)
+{
+    // Avoid indexing out of bounds (and crashing).
```

Comment:
In practice, ZBinBitArray and TileBuffer contain the same data (bitfield) and have the same index calculation (with single dimension parameter for indexing). We should use a single buffer for both and share the functions. Though?

Reply:
We should reduce the number of resources if possible. We already have too many resources bound at a time. They end up taking up register space.

Reply:
We can do that, but I think we will need separate offset/count tables, because TileBufferIndex can not be compressed.

Reply:
My suggestion is to merge zBinBuffer with your new buffer.

Reply:
this ^